torch/jit/_script.py

```diff
@@ -1221,6 +1221,14 @@ def _get_overloads(obj):
     if uncompiled_overloads is None:
         return existing_compiled_fns
 
+    if obj in uncompiled_overloads:
```

Comment:
I am not super familiar with this code. Why does this check imply that there is no definition for the function?

Reply:
If user put "@overload def foo():" as the last toplevel item for overload `foo` i.e. no definition, then jit compiler will recognize that as the "definition" since it's the last defined item in python with that name, so that decl function will appear as `obj` here.